CONTRIB.md

```diff
@@ -1,33 +1,128 @@
+        - [CAPP Camp Git I and Git II](https://uchicago-capp30121-aut-2022.github.io/capp-camp/s2-git-i/)
+        - [GitHub Feature Branches](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+        - [GitHub Pull Requests](https://www.atlassian.com/git/tutorials/making-a-pull-request)
+
+2. Clone the `data-visualization` repo locally.
+
+    - Learn more about cloning [here](https://uchicago-capp30121-aut-2022.github.io/capp-camp/s3-git-ii/1-working-from-multiple-locations.html#using-git-clone)
+
+3. Download a code editor
+
+    - If you do not have a strong opinion about which code editor to use, download [Visual Studio Code](https://code.visualstudio.com/) (often just called VS Code). If you do have a strong opinion, feel free to use your preferred choice.
+        - learn more about code editors generally [here](https://missing.csail.mit.edu/2020/editors/)
+        - learn more about VS Code [here](https://uchicago-capp30121-aut-2022.github.io/capp-camp/s1-linux/2-vs-code-layout.html)
+
+4. Download node which will allow you to run Javascript, the programming language we use to write our visualizations in:
+
+    - [node.js](https://nodejs.org/en) `v20.9.0`
+        - To install the correct version, open the terminal and run:
+            1. Install [Node Version Manager](https://github.com/nvm-sh/nvm) with `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash` . If you are getting error "should not accept aliases with slashes" (likely if you are on Windows) try using `nvm-setup.exe` from the [Windows release site](https://github.com/coreybutler/nvm-windows/releases).
+            2. Install the correct version of Node `nvm install 20.9.0`. If you are getting error of nvm not recognized then run `source ~/.nvm/nvm.sh` and then run above again.
+            3. Set this version as the default by running `nvm alias default 20.9.0` and `nvm use 20.9.0`
+            4. Confirm the correct version is installed by running `node -v`. This should return `v20.9.0`
+
+5. After you have cloned the repo and downloaded Node, go to the base folder of the `data-visualizations` repository and run the command `npm ci` to install all package dependencies. You will only need to do this once
+
+### Tools and Skills to Review
+
+Programming is a skill that you learn by doing. There is no expectation that someone fully understands a programming language or package before they can start creating things. Much of the process is about learning as you go and building a muscle of problem solving and debugging as you go.
+
+With that being said, use the resources below as a starting place and reference source. Aim to learn just enough to jump into a project and learn the rest as you go.
+
+-   We primarily use [`HighCharts`](https://www.highcharts.com/), a JavaScript library, to build visualizations. Each HighChart graphic is embedded on a simple HTML page that is linked in its parent story.
```

Comment:
🔬  We don't line breaks between the main points and subpoints throughout the rest of the documentation. I'd prefer to keep it that way, but if we are going to add the line break then it should be consistent through the rest of the documentation as well.

Reply:
this doenst make a difference in the render of the md as far i can tell